README.md

```diff
@@ -194,6 +194,17 @@ Default: `10000` (10 seconds)
 aicommits config set timeout=20000 # 20s
 ```
 
+#### length
+The maximum number of tokens to be generated for the commit message.
+
+Default: `20` (80 characters)
```

Comment:
For ease of use, this should accept character length, and we can convert that into tokens for them.

Reply:
How do you suggest to convert character length (int) to convert to number of tokens?

Reply:
Ah true. Yeah using TikToken for a baseline is fine. I think maybe add 5~10 characters for padding.

We should also include in the prompt to limit the message to under x characters.

Reply:
We can also use this logic I guess `(Number of chars + 5) / 4`. This will pretty much result the same as TikToken to get the number of tokens.

https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them